AssetManagementTestProject/TestCases/US308_DisableUserTest.cs

```diff
@@ -0,0 +1,40 @@
+﻿using AssetManagementTestProject.PageObj;
+using AssetManagementTestProject.TestSetup;
+using NUnit.Framework;
+
+namespace AssetManagementTestProject;
+
+[TestFixture]
+public class US308_DisableUserTest : NUnitWebTestSetup
+{
+
+    protected ManageUserPage? ManageUserPage;
+
+    public void TC02_AdminCanCancelDisableAction()
+    {
+        ManageUserPage = new ManageUserPage();
+        LoginPage?.Login(Constant.ADMIN_USERNAME_HN, Constant.ADMIN_PASSWORD);
+        DriverBaseAction?.WaitToBeVisible(HomePage.HeaderHomePage);
+        Asserter?.AssertElementIsDisplayed(HomePage.HeaderHomePage);
+        ManageUserPage.GoToUserList();
+        ManageUserPage.InputSearch(NewUser.Data.StaffCode);
+        DriverBaseAction?.WaitToBeVisible(ManageUserPage.TableData);
+        Asserter?.AssertElementIsDisplayed(ManageUserPage.TableData);
```

Comment:
You made a lot of unnecessary assertions without specifying why.

Reply:
In this step, I want to verify that when I search for the staff code, it will appear

Reply:
What does "it" mean here? And even so, please add a comment for specification. Many lines of code without a comment for explanation are confusing. 
I made a few comments in my PR 307, you can check out
![image](https://user-images.githubusercontent.com/79431746/207068587-227db9a5-1e3f-40af-8bd9-af01616f6c79.png)

Reply:
Thank you